app/serializers/v1/task_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module V1
+  class TaskSerializer < ActiveModel::Serializer
+
+    attributes :id, :name, :description, :state
```

Comment:
With your routes you need the `project_id` to get or create or interact with the tasks, so I think it's reasonable it is not serialized into the task. It might be worth considering having it though. Definitely a toss up and an ok choice.